pkg/cmd/helm/release/release.go

```diff
@@ -322,9 +322,6 @@ func (o *Options) OCIRegistry(repoURL, chartDir, name string) error {
 		c = &cmdrunner.Command{
 			Dir:  chartDir,
 			Name: o.HelmBinary,
-			Env: map[string]string{
-				"HELM_EXPERIMENTAL_OCI": "1",
-			},
 			Args: []string{"registry", "login", repoURL, "--username", o.RepositoryUsername, "--password", o.RepositoryPassword},
```

Comment:
How does this work with the registry json file?

Reply:
The flag is only required with helm before 3.8 https://helm.sh/docs/topics/registries/#enabling-oci-support-prior-to-v380

Reply:
ok @ankitm123 ?